x/data/server/testsuite/suite.go

```diff
@@ -92,6 +92,14 @@ func (s *IntegrationTestSuite) TestScenario() {
 	s.Require().Equal([]string{s.addr1.String()}, queryRes.Signers)
 	s.Require().Empty(queryRes.Content)
 
+	// query data by signer
+	bySignerRes, err := s.queryClient.BySigner(s.ctx, &data.QueryBySignerRequest{
+		Signer: s.addr1.String(),
+	})
+	s.Require().NoError(err)
+	s.Require().NotNil(bySignerRes)
+	s.Require().Contains(bySignerRes.Cids, cidBz)
```

Comment:
I wonder, why do we want to use the long calls, rather than declaring a variable at the top of the function:
```
require := s.Require()
```

Reply:
🤷🏻‍♂️